.nvmrc

```
@@ -1 +1 @@
-v18
+20
```

```suggestion
v20
```

is just `20` also valid?

---

Yeah :), node doesn't care